registry/registry.go

```diff
@@ -213,6 +213,9 @@ func (e ErrNoSupport) Error() string {
 func ContinueOnError(err error) bool {
 	switch v := err.(type) {
 	case errcode.Errors:
+		if len(v) < 1 {
```

Comment:
I'd prefer `== 0`. len func can't return negative values and `== 0` better shows your intentions.